boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+.. _ttgo_t8c3:
+
+Lilygo ttgo t8-c3
+#################
+
+Overview
+********
+
+Lilygo ttgo t8-c3 is an IoT mini development board based on the
+Espressif ESP32-C3 WiFi/Bluetooth dual-mode chip.
+
+It features the following integrated components:
+
+- ESP32-C3 chip (160MHz single core, 400KB SRAM, Wi-Fi)
+- on board antenna and IPEX connector
+- USB-C connector for power and communication
+- JST GH 2-pin battery connector
+- LED
+
+.. figure:: img/ttgo_t8c3.webp
+   :align: center
+   :alt: ttgo t8-c3
+
+   Lilygo ttgo t8-c3
+
+Functional Description
+**********************
+This board is based on the ESP32-C3 with 4MB of flash, WiFi and BLE support. It
+has an USB-C port for programming and debugging, integrated battery charging
+and an on-board antenna. The fitted U.FL external antenna connector can be
+enabled by moving a 0-ohm resistor.
+
+Connections and IOs
+===================
+
+The ttgo t8-c3 board configuration supports the following hardware features:
+
++-----------+------------+------------------+
+| Interface | Controller | Driver/Component |
++===========+============+==================+
+| PMP       | on-chip    | arch/riscv       |
++-----------+------------+------------------+
+| INTMTRX   | on-chip    | intc_esp32c3     |
++-----------+------------+------------------+
+| PINMUX    | on-chip    | pinctrl_esp32    |
++-----------+------------+------------------+
+| USB UART  | on-chip    | serial_esp32_usb |
++-----------+------------+------------------+
+| GPIO      | on-chip    | gpio_esp32       |
++-----------+------------+------------------+
+| UART      | on-chip    | uart_esp32       |
++-----------+------------+------------------+
+| I2C       | on-chip    | i2c_esp32        |
++-----------+------------+------------------+
+| SPI       | on-chip    | spi_esp32_spim   |
++-----------+------------+------------------+
+| TWAI      | on-chip    | can_esp32_twai   |
++-----------+------------+------------------+
+
+.. figure:: img/ttgo_t8c3_pinout.webp
+   :align: center
+   :alt: ttgo t8-c3 pinout
+
+   Lilygo ttgo t8-c3 pinout
```

Comment:
i would suggest dropping this and the associated image file. It's not worth adding 80K to the Git repo as people can always refer to the board's official docs